replica/server.go

```diff
@@ -157,9 +159,9 @@ func (s *Server) Create(size int64) error {
 
 func (s *Server) Open() error {
 	s.Lock()
-	defer s.Unlock()
 
 	if s.r != nil {
+		s.Unlock()
```

Comment:
why these changes?

Reply:
Ok, earlier there was go routine introduced so defer was not helpful. Will revert these changes.